app/javascript/components/public_health/PatientsTable.js

```diff
@@ -496,6 +497,11 @@ class PatientsTable extends React.Component {
     return moment(endOfMonitoring, 'YYYY-MM-DD').format('MM/DD/YYYY');
   };
 
+  formatDOB = data => {
+    const rowData = data.rowData;
```

Comment:
do we need to declare this or can we just change the `formatDateOfBirth` call?

Reply:
We cannot change the formatDateOfBirth call, no: the function is passed into CustomTable as an argument, which calls it on the rowData attribute itself. Changing it would require reworking not only CustomTable but every single table formatting function in the entire application.

Reply:
I meant simply doing: `return formatDateOfBirth(data.rowData.dob, data.rowData.id);`

Reply:
Ahhhh, my mistake. That's doable, sure